### ISSUE TITLE
jsx not rendering

### ISSUE THREAD

**Issue by renenadorp** [observablehq/framework#1444]
Hi,
I'm following this example from the Framework documentation:
```jsx
function Greeting({subject}) {
  return <div>Hello, <b>{subject}</b>!</div>
}
```

and then

```
display(<Greeting subject="JSX" />);
```

Unfortunately jsx code blocks are not executed, but are returned as code blocks:
<img width="1085" alt="image" src="https://github.com/observablehq/framework/assets/10479316/8f3aa22b-964e-40c3-a205-5ae401b083a1">

Any idea what I could be doing wrong? 

The complete markdown file is added here below:

[test2.md](https://github.com/user-attachments/files/15749648/test2.md)

**Comment by mbostock** [comment]
This feature is marked as prerelease. Unless you are using the top-of-trunk version by cloning this repo and building Framework yourself, it won’t work. It will be released soon as version 1.9.0. Glad you’re excited to try it out!

**Comment by mbostock** [comment]
[Framework 1.9.0](https://github.com/observablehq/framework/releases/tag/v1.9.0) is now released! You can create a new project as:

```sh
npm exec @observablehq/framework@latest create
```
